README.md

```diff
@@ -109,6 +109,50 @@ Upgrading from older versions of Hive
   different database for your MetaStore you will need to provide
   your own upgrade script.
 
+Using Iceberg REST catalog
```

Comment:
It's strange that adding some properties into README.md.

Maybe we should add these stuff into hive website, such as https://hive.apache.org/docs/latest/apache-hive-4-0-x_282102245/